### ISSUE TITLE
Change class name profile to user

### ISSUE THREAD

**Issue by Apipilikas** [Apipilikas/TowardsGreen#9]
The profile class needs to be renamed to user, so that it will be more clear its use. Also, I personally find it useless to call it that way (and wrong in terms of class modeling).

--
**Disclaimer**

_The reason of the need to change most of the assets:_
Many of the issues that have been created recently are supposed to change main assets of the app. This app was created by a team of three people for a universtity project, whom I thank them for their help and collaboration. That means that the main assets of the app were seperated and allocated to each one of us. The communication wasn't the best one, so that we would manage to create a well-GUI-integrated app. Also, it is acceptable and fine that we don't set the same goals and we don't have the same expectations for the app. 

Bottom line: I would like to turn this good app to an even better, well-working/well-integrated and beautiful one.
